pkg/shell/base_index.js

```diff
@@ -46,8 +45,7 @@ function Frames(index, setupIdleResetTimers) {
     self.iframes = { };
 
     function remove_frame(frame) {
-        $(frame.contentWindow).off();
```

Comment:
I can't actually see which event handlers were attached to contentWindow. Was that just a leftover, or does/did it happen through aliasing? The [docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/remove) don't say anything about auto-cleaning event handlers on removal.

Reply:
We actually do setup some event listeners:
https://github.com/cockpit-project/cockpit/blob/main/pkg/shell/base_index.js#L97
and https://github.com/cockpit-project/cockpit/blob/main/pkg/shell/base_index.js#L552
But `element.remove()` should also disconnect all listeners. (Not that you would get any of these events on element that was removed from the tree anyway).